ocs_ci/ocs/ui/infra_ui.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+
+
+logger = logging.getLogger(__name__)
+
+
+class InfraUI(PageNavigator):
+    """
+    InfraUI Class
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        ocp_version = get_ocp_version()
+        self.infra_loc = locators[ocp_version]["infra"]
+        if config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM:
+            self.storage_class = "thin_sc"
+        elif config.ENV_DATA["platform"].lower() == constants.AWS_PLATFORM:
+            self.storage_class = "gp2_sc"
```

Comment:
same logic is already part of https://github.com/red-hat-storage/ocs-ci/blob/f120044486631f49133c9f3a137842673d765a1c/ocs_ci/deployment/deployment.py#L675

Can you please create a function out of these checks to be used by both places?

Reply:
yes, I will create `Common Function Class`
![image](https://user-images.githubusercontent.com/61982127/109524772-8d1da200-7ab9-11eb-9931-5d6e2d2e58fc.png)